### PR TITLE
Added Updated command

### PR DESCRIPTION
**Pull request by vishal-bhola**
I have added how to **upgrade tensorflow version** in .Readme file

### CODE REVIEW

**Comment by gbaned** [comment]
@VersatileVishal please sign CLA

**Comment by VersatileVishal** [comment]
Done - I have signed CLA.

**Comment by VersatileVishal** [comment]
@googlebot I signed it!

File: README.md [hunk 1]
```diff
@@ -51,6 +51,16 @@ Windows)*:
 $ pip install tensorflow-gpu
 ```
 
+## Upgrade Tensorflow
+
+
+To update Tensorflow to the latest version(currently 2.0)
```
Comment (tomasyany) [inline README.md]: NIT: 
```suggestion
To update Tensorflow to the latest version (currently 2.0)
```

Comment (VersatileVishal) [reply README.md]: There is no major changes.

**Comment by VersatileVishal** [comment]
@mihaimaruseac could you please review my PR

File: README.md [hunk 2]
```diff
@@ -51,6 +51,16 @@ Windows)*:
 $ pip install tensorflow-gpu
 ```
 
+## Upgrade Tensorflow
```
Comment (mihaimaruseac) [inline README.md]: No need to create a new header, no need to mention current version.

How about

```
To update TensorFlow to the latest version, add `--upgrade` flag to the above commands.
```

I don't really think we need to spend much time on this as people can easily get to the `--upgrade` flag of pip elsewhere. The purpose of this section is not to provide a full `pip` manual, but only to list the pip packages that TF provides.

**Review by mihaimaruseac — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (VersatileVishal) [reply README.md]: Done as per your request

**Review by mihaimaruseac — APPROVED** [review]

**Comment by VersatileVishal** [comment]
@mihaimaruseac  can you tell me who is going to merge my PR.

**Comment by VersatileVishal** [comment]
@tomasyany  i have made changes. Please review my PR.

File: README.md (hunk 1 above)
Comment (VersatileVishal) [reply README.md]: Changes as per new request of @mihaimaruseac  is done please review my PR.

**Comment by mihaimaruseac** [comment]
There is going to be an automated process. Tests are going to be run, if all is green PR is going to be merged automatically.

**Comment by VersatileVishal** [comment]
Can you please tell me how long will it take.

**Comment by mihaimaruseac** [comment]
Please have some patience. The CI process is long but should not take more than a day if everything goes smoothly

**Comment by VersatileVishal** [comment]
Thanks for the information really helpful.

**Comment by VersatileVishal** [comment]
@mihaimaruseac  it says 1 errored, 2 failing and 6 successful checks. So, what i have to do now.

**Comment by mihaimaruseac** [comment]
It should be merged soon(ish)

**Pull request merged**